src/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Henok's Portfolio</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header class="header container">
+      <!-- Logo -->
+      <div class="header__logo">My Logo</div>
+      <!-- Navigation -->
+      <nav class="header__nav">Desktop Navigation</nav>
+
+      <!-- Hamburger Menu -->
+      <img
+        class="header__hamburger-menu"
+        src="assets/icons/hamburger.svg"
+        alt="Hamburger menu icon"
+      />
```

Comment:
- [ ] I suggest to ensure that the humburger icon is a `nav` tag. If you notice, the humberger always have the links which are used to navigate into the website to improve the user experience. In addition this is for [semantic reasons](https://www.w3schools.com/html/html5_semantic_elements.asp) ☺️.

Reply:
@AlphaNtihinduka Thank you for your suggestions. I also hope you found my notes regarding the Webhint linter in my submission. I was expecting some answers regarding that.